src/StatusQ/Components/StatusChatListCategory.qml

```diff
@@ -41,7 +41,11 @@ Column {
         showMenuButton: showActionButtons && !!statusChatListCategory.popupMenu
         highlighted: statusChatListCategory.dragged
         sensor.onClicked: {
-            if (sensor.enabled) {
+            if (mouse.button === Qt.LeftButton) {
+                statusChatListCategory.opened = !opened
+                mouse.accepted = true
```

Comment:
Don't want to spoil the party but the Qt docs say:

> When handling this signal, changing the [accepted](https://doc.qt.io/qt-5/qml-qtquick-mouseevent.html#accepted-prop) property of the mouse parameter has no effect, unless the [propagateComposedEvents](https://doc.qt.io/qt-5/qml-qtquick-mousearea.html#propagateComposedEvents-prop) property is true.

Reply:
Thanks for the heads up. Just tested again and it doesn't change anything. It is still ignored and all events are propagated from all the children for those controls regardless of `propagateComposedEvents`/`accepted` and `preventStealing` combination. It looks like something breaks the `MouseArea`s expected behaviour and in that mix of controls in controls and parenting/enablind disabling of mouse areas workaround is hard to tell which one.

Reply:
@stefandunca yeah, avoid nested MouseAreas :) Try with TapHandler instead, it should work

Reply:
@caybro I was referring to that. I tried it and it makes no difference, see the comment in PR: https://github.com/status-im/status-desktop/issues/6733#issuecomment-1206614512